osc-export/pom.xml

```diff
@@ -496,12 +496,17 @@
 										<exclude name="opt/vmidc/bin/libts-jni.so" />
 										<exclude name="opt/vmidc/bin/quartz.properties" />
 										<exclude name="opt/vmidc/bin/vmidcServer.conf" />
-										<exclude name="opt/vmidc/bin/vmidcKeyStore.jks" />
+
+                                        <!-- We might be upgrading from the revision --> 
+                                        <!-- before or after the truststore merge -->
+                                        <exclude name="opt/vmidc/bin/osctrustore.jks" />
```

Comment:
Correct, per discussion.

Reply:
you probably dont need the excludes for 
opt/vmidc/bin/vmidcKeyStore.jks and opt/vmidc/bin/vmidctruststore.jks anymore since they should not be there in first place